angular-hometask/src/app/modules/courses-page/entities/validators/authors-list-size.directive.ts

```diff
@@ -0,0 +1,8 @@
+import { AbstractControl } from '@angular/forms';
+
+export function authorsListValidator(control: AbstractControl): { [key: string]: any } | null {
+  if (control.value && control.value.length > 0) {
```

Comment:
control.value && control.value.length > 0 ? null : { listSize: { value: control.value } }

Reply:
On top of that, you mustn't return a specific value, just return listSize: true, and then you can show specific message for user.

Reply:
516f5acd33596cb71d74107c1c926b967a830522